README.md

```diff
@@ -12,7 +12,38 @@
 
 ## Note on deploying the website
 
-Automatic builds have not yet been implemented. 
+* 1: Add mimic-production to remote
+After the pull request of the new material is merged into the main branch of mimic-website, pull down the latest version of main to your local repo. Run the following command after filling in the IP address for server Heimdallr:
+ 
+`git remote add mimic-production webuser@<IP address for Heimdallr>:/home/webuser/mimic-website.git`
+ 
+* Optional: Check if the command above ran correctly,
+Run command: `git remote -v`
+should return:
+Mimic-production webuser@<IP address for Heimdallr>:/home/webuser/mimic-website.git (fetch)
+Mimic-production webuser@<IP address for Heimdallr>:/home/webuser/mimic-website.git (push)
+origin   https://www.github.com/MIT-LCP/mimic-website (fetch)
+origin   https://www.github.com/MIT-LCP/mimic-website (push)
+ 
+* 2: Sshuttle into the production server
+ 
+Run the following command after filling in the  LCP username and IP address: `alias sshcsail="sshuttle -r <LCP username>@heimdallr.csail.mit.edu <IP address for Heimdallr>/24"`
+ 
+Run command: `sshcsail`
+ 
+Should return:  “client: Connected” after you enter your password
+ 
+* 3: Push development
+ 
+In another terminal from your mimic-website repo:
+ 
+Run command: `git push mimic-production`
+
+
```

Comment:
I thought you were going to add an "Expected errors" section. I think we can wait to fix the errors until someone has some extra time. If you are getting errors and the deployment is still working, I think it's best to mention them here though.

Reply:
I see your update but it's better to give some specific keywords from the expected errors that we are getting. Is this possible?